src/index.ts

```diff
@@ -109,11 +109,12 @@ const createPrismaMock = <P>(
     return joinfield;
   };
 
-  // @ts-ignore
```

Comment:
If you use [`@ts-expect-error`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#ts-ignore-or-ts-expect-error) it will raise when there _isn't_ an error message there